force-app/main/default/lwc/earnedCredentials/earnedCredentials.html

```diff
@@ -0,0 +1,50 @@
+<template>
+	
+	
+	<div  style="    border-radius: 5px;
```

Comment:
this should be moved to css file

Reply:
this is not fixed yet

Reply:
this is not my component , i didn't touch EarnedCredentials component

Reply:
updated code, please review.